src/lib/exporter/streamingDecoder.ts

```
@@ -491,23 +499,29 @@ export class StreamingVideoDecoder {
-			throw new Error(
-				`Video decode ended early at ${decodedAtLabel} (needed ${requiredEndSec.toFixed(3)}s).`,
-			);
-		}
-	}
-
+	const isWindows = typeof navigator !== "undefined" && /Windows/.test(navigator.userAgent);
+
+if (shouldFailDecodeEndedEarly({
+    cancelled: this.cancelled,
+    lastDecodedFrameSec,
+    requiredEndSec,
```

Biome (2.4.10)

[error] The requiredEndSec variable is undeclared.

(lint/correctness/noUndeclaredVariables)